src/sentry/integrations/vercel/integration.py

```diff
@@ -76,6 +77,73 @@ def get_organization_config(self):
 
         return fields
 
+    def update_organization_config(self, mappings):
+        # mappings = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        sentry_project_id = mappings["project_mappings"][0][0]
+        vercel_project_id = mappings["project_mappings"][0][1]
+        org = Project.objects.filter(id=sentry_project_id).get()
+        sentry_project = Project.objects.filter(id=sentry_project_id).get()
+        sentry_project_dsn = ProjectKey.objects.filter(project=sentry_project).get()
```

Comment:
Are you sure this will always give us the right DSN?

Reply:
good point, what do we want to do if there are multiple though? would checking that they're active be enough?

Reply:
@ceorourke I would check the logic that's used in the view for the UI where we ask the user to copy the DSN.